docker-run-sync-rapid-pro-to-engagement-db.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+set -e
+
+PROJECT_NAME="$(<configurations/docker_image_project_name.txt)"
+IMAGE_NAME=$PROJECT_NAME-rapid-pro-to-engagement-db
+
+while [[ $# -gt 0 ]]; do
+    case "$1" in
+        --incremental-cache-path)
+            INCREMENTAL_MODE=true
+            INCREMENTAL_CACHE_PATH="$2"
```

Comment:
I think in dockerised mode this should be a volume instead (and then we automatically create the appropriate sub-directory in the volume in this script)